examples/sample-pub/publisher.js

```diff
@@ -104,6 +104,24 @@ function startFlowAuto() {
     });
     return;
   }
+
+  if (flow == 'smartbutton') {
+    whenReady(function(subsciptions) {
+      var subs = subsciptions;
+      whenDemoReady(function() {
+        var smartButtonContainerDiv =
```

Comment:
It's no longer a container, so let's also remove "container" from the name. And also "div".

Reply:
Done